media/css/products/vpn/components/block.scss

```diff
@@ -0,0 +1,371 @@
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/fonts';
+$image-path: '/media/protocol/img';
+
+@import '../../../../protocol/css/includes/lib';
+@import '../includes/lib';
+
+//* -------------------------------------------------------------------------- */
+// Content Block
+
+.vpn-content-block {
+    @include flexbox;
+    @include align-items(center);
+    -moz-outline-radius: $border-radius-sm;
+    border-radius: $border-radius-sm;
+    color: $color-black;
+    margin-bottom: $layout-lg;
+    outline: 4px solid $color-black;
+    overflow: visible;
+    padding: 0;
+    position: relative;
+    z-index: 0;
+
+    &:after {
+        background: $color-marketing-gray-20;
+        border-radius: $border-radius-sm;
+        content: '';
+        display: block;
+        height: 100%;
+        left: 0;
+        position: absolute;
+        top: 0;
+        width: 100%;
+        z-index: -1;
+    }
+
+    .vpn-content-block-container {
+        margin: 0 auto;
+        padding: $spacing-lg $spacing-xl;
+        width: 100%;
+    }
+
+    .vpn-content-block-heading {
+        @include text-title-md;
+        margin-bottom: $spacing-xl;
+    }
+
+    &.t-highlight {
+        color: $color-black;
+        outline: 4px solid $color-vpn-brand-purple;
+
+        .vpn-content-block-container {
+            background-color: $color-white;
+        }
+
+        &:after {
+            background: #c4c6fc;
+        }
+    }
+
+    &.t-dark {
+        color: $color-white;
+        outline: none;
+
+        .vpn-content-block-container {
+            background-color: transparent;
+        }
+
+        &:after {
+            background: $color-vpn-brand-purple;
+        }
+    }
+
+    @media #{$mq-md} {
+        margin-bottom: $layout-xl;
+
+        &:after {
+            left: $spacing-xl;
+            top: $spacing-xl;
+        }
+
+        .vpn-content-block-container {
+            padding: ($spacing-2xl * 2) $spacing-2xl;
+        }
+
+        &.t-dark {
+            outline: 4px solid $color-black;
+        }
+
+        .l-columns-two {
+            @include clearfix;
+            position: relative;
+
+            .l-column-first {
+                @include bidi(((float, left, right),));
+                width: calc(50% - #{$spacing-xl * 2});
+            }
+
+            .l-column-last {
+                @include bidi(((float, right, left),));
+                width: calc(50% - #{$spacing-xl * 2});
+            }
+        }
+    }
+}
+
+//* -------------------------------------------------------------------------- */
+// Content Block with Image
+
+.vpn-content-media {
+    margin: 0 auto $spacing-xl;
+    overflow: hidden;
+    position: relative;
+    text-align: center;
+    z-index: 0;
+
+    .vpn-content-media-image {
+        img {
+            height: auto;
+            max-width: 480px;
```

Comment:
Should this be a protocol token? `$screen-sm`

Reply:
This value is matching the specific image size, so I'm not sure a screen-sizing token makes sense